OpenOversight/tests/routes/test_user_api.py

```diff
@@ -266,9 +270,11 @@ def test_admin_can_resend_user_confirmation_email(mockdata, client, session):
 
 
 def test_register_user_approval_required(mockdata, client, session):
-    current_app.config["APPROVE_REGISTRATIONS"] = True
-    with current_app.test_request_context():
-        diceware_password = "operative hamster perservere verbalize curling"
+    BaseConfig.APPROVE_REGISTRATIONS = True
+    with current_app.test_request_context(), patch.object(
+        GmailClient, "__new__", return_empty_object
+    ):
```

Comment:
Using the `return_empty_object` function from `utils`  here and lower in the file.